frontend-react/src/components/Table/ColumnData.tsx

```diff
@@ -101,7 +101,12 @@ export const ColumnData = ({
             return action();
         };
         return tableData(
-            <Button type="button" unstyled onClick={doAction}>
+            <Button
+                className="font-mono-2xs line-height-alt-4"
```

Comment:
`font-mono-2xs` is the chosen font for our table components, currently, and `line-height-alt-4` overrides the override that `usa-button` class causes in that component. The proper line height is `1.6` for table text.

<img width="168" alt="Screen Shot 2023-01-10 at 12 20 54 PM" src="https://user-images.githubusercontent.com/4022304/211619442-5c7b87c4-75e1-4fb5-82ab-b370dac07142.png">
<img width="224" alt="Screen Shot 2023-01-10 at 12 21 28 PM" src="https://user-images.githubusercontent.com/4022304/211619443-d9142fc9-1c1b-40a6-833f-8845596ee3b1.png">